lib/ansible/modules/cloud/amazon/elb_target_group.py

```diff
@@ -836,6 +835,9 @@ def main():
                               ]
                               )
 
+    if module.params.get('target_type') is None:
```

Comment:
I've no other idea to handle this. Any idea about this @felixfontein ?

Reply:
maybe we don't need any default value?

Reply:
Not having any default would cause failures for folks who have been relying on default=instance with state=present, we'd need a deprecation cycle to do this.  Logically I think we only need a default if state=present though, so we could put this in the if block at ln843.

Reply:
I'm not sure if I should keep this here open, because of https://github.com/ansible/ansible/pull/61256?

Reply:
Ah ok I knew the issue seemed familiar but I thought that one had been a different module. There's several things going on in that PR though, @tremble what do you think about landing the fix for `elb_target_group` requiring module args when `state=absent` in here, rather than your fix in 61256?

Reply:
I wasn't sure if manipulating module.params[] was "the done thing".  I have no problem with this approach rather than the approach I took.